admin_solutions/user_role_mgmt.adoc

```diff
@@ -0,0 +1,550 @@
+usage:
+
+----
+$ oc get quota
+----
+
+However, this would not tell you what is actually being consumed.
+To determine what is actually being consumed, use the `oc describe` command:
+
+----
+$ oc describe quota <quota-name>
+----
+
+Alternatively, you can set up
+link:../install_config/cluster_metrics.html[cluster metrics] for more detailed statistics.
+
+[[determine-default-user-roles]]
+== Determining Which Roles Users Get by Default
+
+When a user first logs in, there is a default set of permissions that is
+applied to that user. The scope of permissions that a user can have is
+controlled by the
+link:../architecture/additional_concepts/authorization.html#roles[various types]
+of
+link:../admin_guide/manage_authorization_policy.html[roles within OpenShift]:
+
+- `ClusterRoles`
+- `ClusterRoleBindings`
+- `Roles` (project-scoped)
+- `RoleBindings` (project-scoped)
+
+You may want to modify the default set of permissions. In order to do this, it's important to understand the default groups and roles assigned, and to be aware of the roles and users bound to
+link:../admin_solutions/user_role_mgmt.html#view-roles-users-project[each project] or
+link:../admin_solutions/user_role_mgmt.html#view-roles-users-cluster[the entire cluster].
+
+=== Leveraging Default Groups
+
+There are special groups that are assigned to users. You can target users with these groups, but you cannot modify them. These special groups are as follows:
+
+[options="header"]
+|===
+
+|Group |Description
+
+|`system:authenticated`
+|This is assigned to all users who are identifiable to the API. Everyone who is not `system:anonymous` (the user) is in this group.
+
+|`system:authenticated:oauth`
+|This is assigned to all users who have identified using an oauth token issued by the embedded oauth server. This is not applied to service accounts (they use service account tokens), or certificate users.
+
+|`system:unauthenticated`
+|This is assigned to users who have not presented credentials. Invalid credentials are rejected with a 401 error, so this is specifically users who did not try to authenticate at all.
+|===
+
+You may find it helpful to target users with the special groups listed above. For example, you could share a pod with all users by granting `system:authenticated` access to the pod.
+
+The "default" permissions of users are defined by which roles are bound to the
+`system:authenticated` and `sytem:authenticated:oauth` groups. As mentioned
+above, you are not able to modify membership to these groups, but you can
+link:../admin_guide/manage_authorization_policy.html#managing-role-bindings[change the roles bound to these groups]. For example, to bind a role to the `system:authenticated` group for all projects in the cluster:
+
+----
+$ oadm policy add-cluster-role-to-group <role> system:authenticated
+----
+
+Currently, by default the `system:authenticated` and `sytem:authenticated:oauth`
```

Comment:
Hmm, I'm not too scripty, but I bet @vikram-redhat could do it (but he's a busy guy).